src/react-aria-modal.js

```diff
@@ -166,9 +167,9 @@ let Modal = class Modal extends React.Component {
       dialogProps.tabIndex = '-1';
     }
 
-    const childrenArray = [React.DOM.div(dialogProps, props.children)];
+    const childrenArray = [DOM.div(dialogProps, props.children)];
```

Comment:
It just tried it with createElement, it works just fine! Tests okay, examples still working. 🕺